smack-extensions/src/main/java/org/jivesoftware/smackx/message/correct/package-info.java

```diff
@@ -0,0 +1,21 @@
+/**
+ *
+ * Copyright 2016 Fernando Ramirez
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Classes and Interfaces that implement Last Message Correction as defined in XEP-0308.
+ */
+package org.jivesoftware.smackx.message.correct;
```

Comment:
Please change the package name to `message_correct` (if it is possible, but I think it should).